incubator/hnc/internal/validators/hncconfig_test.go

```diff
@@ -165,17 +166,45 @@ func TestNonRBACTypes(t *testing.T) {
 
 func TestPropagateConflict(t *testing.T) {
 	tests := []struct {
-		name         string
-		inNamespaces string
-		allow        bool
+		name            string
+		forest          string
+		inNsesWithAnnot map[string]map[string]string
+		allow           bool
+		errContain      string
 	}{{
-		name:         "Objects with the same name existing in namespaces that one is not an ancestor of the other would not cause overwriting conflict",
-		inNamespaces: "bc",
-		allow:        true,
+		name:   "Objects with the same name existing in namespaces that one is not an ancestor of the other would not cause overwriting conflict",
+		forest: "-aa",
+		inNsesWithAnnot: map[string]map[string]string{
+			"b": map[string]string{},
+			"c": map[string]string{},
+		},
```

Comment:
You've gone from `"bc"` to:

```
map[string]map[string]string{
			"b": map[string]string{},
			"c": map[string]string{},
		},
```

I think this hurts readability. Can you think of a way to make the tests easier to read while only asking for all this complexity when it's really needed?

Reply:
I agree, I replaced it with a variable `empty`, does it look better?